pkg/apis/stork/v1alpha1/snapshotrestore.go

```diff
@@ -47,6 +50,12 @@ type RestoreStatus struct {
 	Status SnapshotRestoreStatusType `json:"status"`
 	// Volumes list of restore inforamtion
 	Volumes []*RestoreVolumeInfo `json:"volumes"`
+	// RestoreVolume map of snapID and volID to restore
+	RestoreVolumes map[string]string `json:"restoreVolumes"`
+	// List of PVC associated with snapshot restore
+	PVCs []*v1.PersistentVolumeClaim `json:"pvcs"`
+	// RestoreType restore type of volumes
+	RestoreType string `json:"restoreType"`
```

Comment:
This shouldn't be in the CRD. This is PX specific. Store this as an annotation in the spec which PX can refer to

Reply:
Actually, it doesn't need to be stored anywhere. We should always get it from the snapshot information